src/app/components/shell/header/main-menu/login-menu/login-menu.js

```diff
@@ -72,13 +73,22 @@ class LoginMenu extends componentType(spec, busMixin) {
                 url: tutorDomain,
                 exclude: () => !this.isTutorUser
             },
+            {
+                label: 'Finish signing up',
+                url: facultySignupStep4,
+                exclude: () => Boolean(
+                    (this.user.groups || []).includes('Student') ||
+                    !this.user.needs_profile_completed ||
+                    !this.user.is_newflow ||
+                    this.user.stale_verification
+                )
+            },
             {
                 label: 'Request instructor access',
-                url: facultyAccessLink,
+                url: reqFacultyAccessLink,
                 exclude: () => Boolean(
                     (this.user.groups || []).includes('Faculty') ||
-                    (this.user.groups || []).includes('Student') ||
-                    this.user.pending_verification
+                    (!this.user.stale_verification && this.user.pending_verification)
```

Comment:
Note: If my logic is right, this should show up for users who are "stale". @RoyEJohnson

Reply:
But at the same time, hide it for users who have not finished signing up

Reply:
It will show up if they're stale _or_ not pending. Is that right? Or should there be a ! on the pending as well, so it would show up if they're stale or pending?

Reply:
Correct, it should show up if they're stale or _not_ pending. Because if they're just pending and not stale, I don't want it to show up; in that case, that means they're in the middle of the signup process and thus I want the "Finish signing up" link to show up instead. 

(The default state from the beginning is pending, and then as they progress through the flow, it will be set to approved or rejected. Unless they are stale after 4 days stuck in pending.)